src/components/Search/Search.js

```diff
@@ -117,8 +117,7 @@ class Search extends React.PureComponent {
   };
 
   handleFilterChange = (ev, value) => {
-    this.input.focus();
-    this.addSearchTerm(ev, value);
```

Comment:
I just realized `addSearchTerm` doesn't use `ev`, maybe you can remove it from there?